plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/CassandraTestingUtils.java

```diff
@@ -237,6 +239,54 @@ public static void createTableAllTypesPartitionKey(CassandraSession session, Sch
         insertTestData(session, table, date, 9);
     }
 
+    public static void createTableAllTypesPushdownPartitionKey(CassandraSession session, SchemaTableName table, Date date)
+    {
+        session.execute("DROP TABLE IF EXISTS " + table);
+
+        session.execute("CREATE TABLE " + table + " (" +
+                " key text, " +
+                " typeuuid uuid, " +
+                " typetinyint tinyint, " +
+                " typesmallint smallint, " +
+                " typeinteger int, " +
+                " typelong bigint, " +
+                " typebytes blob, " +
+                " typedate date, " +
+                " typetimestamp timestamp, " +
+                " typeansi ascii, " +
+                " typeboolean boolean, " +
+                " typedecimal decimal, " +
```

Comment:
I've skipped (in this initial commit) adding the `typedecimal` field to the composed primary key because the test is failing. Apparently in Cassandra 2.2 when comparing `128.0` with `128` there is no match found.

Reply:
Could you file an issue and leave as code comment?